src/taipy/core/scenario/scenario.py

```diff
@@ -233,7 +233,6 @@ def name(self) -> Optional[str]:
         return self._properties.get("name")
 
     @name.setter  # type: ignore
-    @_self_setter(_MANAGER_NAME)
```

Comment:
I don't get why you removed the self setter decorator.

Reply:
Ah that's because it's not needed! But setting/updating a new value in self._properties, the dictionary would already set the entity. Having @_self_setter will only set the entity again without any changes!